Library/Formula/libbpg.rb

```diff
@@ -11,24 +12,38 @@ class Libbpg < Formula
     sha256 "7b8d1585ee9e2de010abfab1aaab419d052cf35d82704587c2f75947d4fc8ee5" => :mavericks
   end
 
-  option "with-x265", "Enable x265 encoder"
-  option "without-jctvc", "Disable built-in JCTVC encoder"
+  option "with-jctvc", "Enable built-in JCTVC encoder - Mono threaded, slower but produce smaller file"
+  option "without-x265", "Disable built-in x265 encoder - Multi threaded, faster but produce bigger file"
```

Comment:
As I say above, it's one of major change between version `0.9.5` and `0.9.6`.
With `0.9.5`, `jctvc ` was the default encoder and `x265` is optional (depends on `x265` formula).
With `0.9.6`, `x265` is the default encoder (do not depends on `x265` formula anymore) and `jctvc` is optional.